dumang_ctrl/dumang/common.py

```diff
@@ -621,11 +685,51 @@ def __init__(self, key, layer_keycodes):
         self.layer_keycodes = {k: v.encode() if isinstance(v, Keycode) else v for k, v in layer_keycodes.items()}
 
     def encode(self):
-        return [self.cmd, 0x01, self.layer_keycodes[1], self.layer_keycodes[2], self.layer_keycodes[3], 0xFF, self.layer_keycodes[0]]
+        return [self.cmd, self.key.encode() + 1, self.layer_keycodes[1], self.layer_keycodes[2], self.layer_keycodes[3], 0xFF, self.layer_keycodes[0]]
```

Comment:
Can you elaborate on this a bit? What's this byte actually for? I wasn't able to decipher it the first time around. Also why the `+ 1`?

Reply:
it's just my attempt to mimic what the official app does.